Mathlib/CategoryTheory/Join/Basic.lean

```diff
@@ -541,6 +541,38 @@ lemma mapIsoWhiskerLeft_inv (H : C ⥤ E) {Fᵣ : D ⥤ E'} {Gᵣ : D ⥤ E'} (
 
 end NaturalTransforms
 
+section mapPairEquiv
+
+variable {E : Type u₃} [Category.{v₃} E]
+  {E' : Type u₄} [Category.{v₄} E']
+
+variable {C D}
+
+/-- Equivalent categories have equivalent joins. -/
+@[simps]
+def mapPairEquiv (Eₗ : C ≌ E) (Eᵣ : D ≌ E') : C ⋆ D ≌ E ⋆ E' where
```

Comment:
I'd think we could find better names here, C/D then E/E' feels confusing, and E_l/E_r is hard to read (I thought the l was a 1 first!)